core/domain/feedback_services_test.py

```diff
@@ -325,15 +326,34 @@ def _close_thread(exp_id, thread_id):
 class EmailsTaskqueueTests(test_utils.GenericTestBase):
     """Tests for tasks in emails taskqueue."""
 
-    def test_create_new_task(self):
+    def test_create_new_batch_task(self):
         user_id = 'user'
-        feedback_services.enqueue_feedback_message_email_task(user_id)
+        feedback_services.enqueue_feedback_message_batch_email_task(user_id)
         self.assertEqual(self.count_jobs_in_taskqueue(), 1)
 
         tasks = self.get_pending_tasks()
         self.assertEqual(
             tasks[0].url, feconf.FEEDBACK_MESSAGE_EMAIL_HANDLER_URL)
 
+    def test_create_new_instant_task(self):
+        user_id = 'user'
+        reference = {
+            'exploration_id': 'eid',
+            'thread_id': 'tid',
+            'message_id': 'mid'
+        }
+        reference_domain = feedback_domain.FeedbackMessageReference(
+            reference['exploration_id'], reference['thread_id'],
+            reference['message_id'])
+
+        feedback_services.enqueue_feedback_message_instant_email_task(
+            user_id, reference_domain)
+        self.assertEqual(self.count_jobs_in_taskqueue(), 1)
+
+        tasks = self.get_pending_tasks()
+        self.assertEqual(
+            tasks[0].url, feconf.INSTANT_FEEDBACK_EMAIL_HANDLER_URL)
```

Comment:
I don't think that it is necessary, it gets cross checked in other tests implicitly.